ocaml/xapi/xapi_cluster_host.ml

```diff
@@ -63,6 +63,18 @@ let create_as_necessary ~__context ~host =
         Client.Client.Cluster_host.create rpc session_id cluster_ref host) |> ignore
   | None -> ()
 
+let resync_host ~__context ~host =
+  create_as_necessary ~__context ~host;
+  match (find_cluster_host ~__context ~host) with
+    | exception (Api_errors.(Server_error(_,_)) as err)
+                        -> raise err
```

Comment:
This is redundant: if you do not catch the exception it will escape on  its own.